src/components/data_submission/NIHAdministrativeInformation.js

```diff
@@ -0,0 +1,134 @@
+import {div, h, h2} from 'react-hyperscript-helpers';
+import { FormFieldTypes, FormField, FormValidators } from '../forms/forms';
+import { nihInstitutions } from './nih_institutions';
+import { isEmpty } from 'lodash/fp';
+import { useState } from 'react';
+
+export const NIHAdministrativeInformation = (props) => {
+  const {
+    initialFormData,
+    onChange,
+    institutions,
+  } = props;
+
+  const [showMultiCenterStudy, setShowMultiCenterStudy] = useState(initialFormData?.multiCenterStudy === true || false);
+  const [showGSRNotRequiredExplanation, setShowGSRNotRequiredExplanation] = useState(initialFormData?.controlledAccessRequiredForGenomicSummaryResultsGSR === false || false);
+  const [gsrNotRequiredExplanation, setGSRNotRequiredExplanation] = useState('');
+
+  return div({
+    className: 'data-submitter-section',
+  }, [
+    h2('NIH Administrative Information'),
+    h(FormField, {
+      id: 'piName',
+      title: 'Principal Investigator Name',
+      validators: [FormValidators.REQUIRED],
+      onChange,
+      defaultValue: initialFormData?.piName,
+    }),
+    h(FormField, {
+      id: 'piInstitution',
+      title: 'Principal Investigator Institution',
+      isRendered: !isEmpty(institutions),
+      validators: [FormValidators.REQUIRED],
+      type: FormFieldTypes.SELECT,
+      selectOptions: institutions,
+      isCreatable: false,
+      selectConfig: {},
+      onChange,
+      defaultValue: initialFormData?.piInstitution,
+    }),
+    h(FormField, {
+      id: 'nihGrantContractNumber',
+      title: 'NIH Grant or Contract Number',
+      validators: [FormValidators.REQUIRED],
+      onChange,
+      defaultValue: initialFormData?.nihGrantContractNumber,
+    }),
+    h(FormField, {
+      id: 'nihICsSupportingStudy',
+      title: 'NIH ICs Supporting the Study',
+      placeholder: 'Institute/Center Name',
+      onChange,
+      type: FormFieldTypes.SELECT,
+      validators: [FormValidators.REQUIRED],
+      defaultValue: initialFormData?.nihICsSupportingStudy,
+      selectOptions: nihInstitutions,
+    }),
```

Comment:
I'm wondering if this needs to be a multi-select. Can we clarify with Jonathan/Pamela? The schema does NOT support that now, so we'd need to update that if so.

Reply:
Just got back clarification: this should be multiselect, but the other one should stay single-value. I updated the ticket in jira to update json schema to include this change as well.